.github/CODEOWNERS

```
@@ -34,3 +34,6 @@
 
 # Contracts
 /packages/contracts-bedrock     @ethereum-optimism/contract-reviewers
+
+# Security docs
+/docs @ethereum-optimism/security-reviewers
```

this `security-reviewers` group is deprecated, we should probably just delete it
```suggestion
/docs @ethereum-optimism/evm-safety
```